presto-geospatial/src/main/java/com/facebook/presto/geospatial/BingTileFunctions.java

```diff
@@ -206,8 +206,8 @@ private static BingTile latitudeLongitudeToTile(double latitude, double longitud
         double y = 0.5 - Math.log((1 + sinLatitude) / (1 - sinLatitude)) / (4 * Math.PI);
 
         int mapSize = mapSize(zoomLevel);
-        int tileX = (int) clip(x * mapSize + 0.5, 0, mapSize - 1);
```

Comment:
When `x * mapSize = 1.9`, should its tileX be `1` or `2`?

Reply:
![screen shot 2017-11-03 at 11 44 52 am](https://user-images.githubusercontent.com/27965151/32382746-74fe69c4-c08c-11e7-8816-3993ae716a73.png)

This image shows tiles at zoom levels 1, 2 and 3. All these tiles should be aligned on a top left corner at (0, 0). But they are not.

Reply:
![screen shot 2017-11-03 at 11 52 26 am](https://user-images.githubusercontent.com/27965151/32383133-831cd116-c08d-11e7-8fd0-d63ee2c3bc67.png)

The point on this chart maps to tile (1, 1) at level 1, but it doesn't map to any tile at higher levels that is part of that tile.